app/controllers/activity_insight_oa_workflow/wrong_file_version_curation_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class ActivityInsightOAWorkflow::WrongFileVersionCurationController < ActivityInsightOAWorkflowController
+  def index
+    @publications = Publication.wrong_file_version.order('wrong_oa_version_notification_sent_at DESC NULLS FIRST')
+  end
+
+  def email_author
+    publications = Publication.wrong_file_version.find(params[:publications])
+
+    FacultyNotificationsMailer.wrong_file_version(publications).deliver_now
+    ActiveRecord::Base.transaction do
+      publications.each do |pub|
+        pub.update_column(:wrong_oa_version_notification_sent_at, Time.current)
+      end
+    end
+    flash[:alert] = "Email sent to #{publications.first.activity_insight_file_uploader.webaccess_id}}"
```

Comment:
oops - one other thing. And this is my fault (I told you the wrong thing). This should be `flash[:notice]` instead of `flash[:alert]`. We're using the former for success messages and the latter for error messages, and I can never remember which one is which off of the top of my head.